File: DotNet/CesiumLanguageWriter/Advanced/ICesiumStringValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a string.
    /// </summary>
    public interface ICesiumStringValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value expressed as a string value.
        /// </summary>
        /// <param name="value">The value.</param>
        void WriteString(string value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumStringValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 *  
 A writer that can write a value as a string.
 

 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumStringValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    *  
    Writes the value expressed as a string value.
    
    

    * @param value The value.
    */
    void writeString(String value);
}